src/command_modules/azure-cli-storage/tests/test_storage_account_scenarios.py

```diff
@@ -2,13 +2,15 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
-
 from azure.cli.testsdk import (ScenarioTest, JMESPathCheck, ResourceGroupPreparer,
-                               StorageAccountPreparer)
+                               StorageAccountPreparer, api_version_constraint)
+from azure.cli.core.profiles import ResourceType
 from .storage_test_util import StorageScenarioMixin
 
 
+@api_version_constraint(ResourceType.MGMT_STORAGE, min_api='2016-12-01')
 class StorageAccountTests(StorageScenarioMixin, ScenarioTest):
```

Comment:
Where is the test for the old version of storage account create?  I don't see any entries with "max_api=..." that would suggest this test exists.